### ISSUE TITLE
Drag and drop does not work

### ISSUE THREAD

**Issue by bberberov** [lwindolf/liferea#118]
Installed 1.10.12 from source on an updated Kubuntu 14.10.

Drag and drop does not work. Cannot reorder feeds and cannot drag items into news bins.  The version in Universe does not work either.

**Comment by lwindolf** [comment]
Dragging items into news bins was never implement, this is not a regression.

I retested DnD without problems. I cannot reproduce the problem. 

Also much more details are needed:

- Could you provide a video/screencast?
- Into which type of folder do you try to drop?
- Is the target a local folder or a folder of a remote source?
- Is the folder empty or does it have contents already?
- Do you get the GtkTreeView insertion marker when hovering the selected feed over the target folder? 
- Can you actually drag the feed you want to move?

**Comment by bberberov** [comment]
I believe I figured out what the problem is.  By the way, I am actually using Kubuntu 14.04 LTS.

Kubuntu 14.04 by default comes with gtk3-engines-oxygen version 1.3.5 as the default GTK 3 theme.  There is an issue with that particular theme that prevents applications like Liferea from handling drag and drop events.  I installed gtk3-engines-unico and a compatible theme, changed it in KDE, and now drag and drop works.

I have not tried Kubutnu 14.10 yet.  It is supposed to have gtk3-engines-oxygen version 1.4.0 and this issue might be fixed there.
